src/Compilers/CSharp/Portable/Symbols/SubstitutedNamedTypeSymbol.cs

```diff
@@ -146,12 +146,9 @@ internal sealed override ImmutableArray<NamedTypeSymbol> GetDeclaredInterfaces(C
             return _unbound ? ImmutableArray<NamedTypeSymbol>.Empty : Map.SubstituteNamedTypes(OriginalDefinition.GetDeclaredInterfaces(basesBeingResolved));
         }
 
-        internal sealed override NamedTypeSymbol BaseTypeNoUseSiteDiagnostics
+        internal sealed override NamedTypeSymbol GetBaseTypeNoUseSiteDiagnostics(bool ignoreNonNullTypesAttribute)
         {
-            get
-            {
-                return _unbound ? null : Map.SubstituteNamedType(OriginalDefinition.BaseTypeNoUseSiteDiagnostics);
-            }
+            return _unbound ? null : Map.SubstituteNamedType(OriginalDefinition.BaseTypeNoUseSiteDiagnostics);
```

Comment:
>BaseTypeNoUseSiteDiagnostics [](start = 80, length = 28)

`GetBaseTypeNoUseSiteDiagnostics(ignoreNonNullTypesAttribute)`

Reply:
Doesn't appear resolved?

---
In reply to: [201403314](https://github.com/dotnet/roslyn/pull/28320#discussion_r201403314) [](ancestors = 201403314)